src/extension.ts

```diff
@@ -1,106 +1,61 @@
-import * as vscode from 'vscode';
-
-import cp = require('child_process');
-import os = require('os');
-import path = require('path');
-import fs = require('fs')
-import ver = require('compare-versions');
-
-const timespan = require("timespan-parser");
-
-const minCLIVersion = "0.2.0";
-
-// this method is called when extension is activated
-export function activate(context: vscode.ExtensionContext) {
-	let disposable = vscode.commands.registerCommand('rhinocode.runInRhino', () => {
-		const activeFile = vscode.window.activeTextEditor?.document.uri.fsPath;
-
-		let configs = vscode.workspace.getConfiguration('rhinocode');
-		let showActiveDocument = configs.get<boolean>("showActiveDocument");
-		let showActiveDocumentPath = configs.get<boolean>("showActiveDocumentPath");
-		let showActiveViewport = configs.get<boolean>("showActiveViewport");
-		let showProcessId = configs.get<boolean>("showProcessId");
-		let showProcessAge = configs.get<boolean>("showProcessAge");
-		let showProcessFullVersion = configs.get<boolean>("showProcessFullVersion");
-
-		if (activeFile != undefined) {
-			const rhinos = getRhinoInstances();
-			if (rhinos.length > 0) {
-				const names = new Map(
-					rhinos.map(r => {
-						let title = r.processName;
-
-						if (showProcessFullVersion)
-							title += ` ${r.processVersion}`;
-						else {
-							const verisonNumbers = r.processVersion.split('.');
-							title += ` ${verisonNumbers[0]}.${verisonNumbers[1]}`;
-						}
-
-						if (showProcessId)
-							title += ` <${r.processId}>`;
-
-						if (showActiveDocument) {
-							if (r.activeDoc.title == undefined) {
-								title += ` - "Untitled"`;
-							}
-							else {
-								title += ` - "${r.activeDoc.title}"`;
-							}
-						}
-
-						if (showActiveDocumentPath) {
-							if (r.activeDoc.title == undefined) {
-								title += ` - Not Saved`;
-							}
-							else {
-								title += ` - @ ${r.activeDoc.location}`;
-							}
-						}
-
-						if (showActiveViewport)
-							title += ` [${r.activeViewport}]`;
-
-						if (showProcessAge) {
-							let age = timespan.parse(`${r.processAge}m`);
-							title += ` (${timespan.getString(age, { abbv: false })})`;
-						}
-
-						return [title, r]
-					})
-				);
-
-				vscode.window.showQuickPick([...names.keys()], { canPickMany: false })
-					.then((v) => {
-						if (v != undefined) {
-							const rhino = names.get(v);
-							const scriptCmd = `"${getRhinoCode()}" --rhino ${rhino?.pipeId} script \"${activeFile}\"`;
-							console.log(scriptCmd)
-							cp.execSync(scriptCmd);
-						}
-					});
-			}
-		}
-		else {
-			vscode.window.showInformationMessage(
-				"There are no active scripts open"
-			);
-		}
-	});
+import * as Vsc  from 'vscode';
+
+import * as Cp   from 'child_process';
```

Comment:
@corbane Why is this better?

Reply:
Ah, yes, I haven't changed that.
It is not better, it is not less good. it is quite possible to put the names back as before.

I write the modules with a capital letter because of `path` (essentially).
`path` is very often a variable `path = "dir/file.txt" `, but it is also very often the module `const path = require ('path') `
I have had bugs several times because of this.

Between `cp` and `Cp` we can identify the variable `cp` of the module `Cp`.